tagsets/helpers.php

```diff
@@ -15,25 +15,26 @@ function support_mail_link() {
 }
 
 function multi_array_sort(&$data, $sortby) {
-    static $sort_funcs = array();
-
+    $sortby1=""; $sortby2=""; $sortby3="";
     if (is_array($sortby)) {
-            $sortby = join(',',$sortby);
-        }
-
-    if (empty($sort_funcs[$sortby])) {
-            $code = "\$c=0;";
-            $sortby_arr=explode(',', $sortby);
-            foreach ($sortby_arr as $key) {
-                $code .= "if ( (\$c = strcasecmp(\$a['$key'],\$b['$key'])) != 0 ) return \$c;\n";
-                }
-            $code .= 'return $c;';
-            $sort_func = $sort_funcs[$sortby] = create_function('$a, $b', $code);
-     } else {
-            $sort_func = $sort_funcs[$sortby];
+        $sortby1 = $sortby[0];
+        if (isset($sortby[1])) $sortby2 = $sortby[1];
+        if (isset($sortby[2])) $sortby3 = $sortby[2];
+    } else {
+        $sortby1 = $sortby;
+    } 
+    global $sorting_var1, $sorting_var2, $sorting_var3;
```

Comment:
OK, using global here is not required (and in general it is not good practice exposing those vars to global context). Just rewrite anonimous function call using `use()` keyword, i.e.:
```
    $sorting_var1=$sortby1; $sorting_var2=$sortby2; $sorting_var3=$sortby3;
    uasort($data, function ($a, $b) use ($sorting_var1, $sorting_var2, $sorting_var3) {
        if ($a[$sorting_var1]!=$b[$sorting_var1] || (!$sorting_var2)) {
            return $a[$sorting_var1]>$b[$sorting_var1];
        } elseif ($a[$sorting_var2]!=$b[$sorting_var2] || (!$sorting_var3)) {
        ...
```
Notice, that using anonymous function requires us to bump minimum PHP supported version to 5.3.

Reply:
Ok, changed to use() syntax as suggested, deleted global declarations. Will change Wiki to PHP 5.3 requirement.